README.md

```diff
@@ -493,6 +499,46 @@ See also:
 
   * [v::key()](#vkeyname) - Validates a specific key of an array
 
+#### v::bank(string $countryCode)
+
+Validates a bank.
+
+```php
+v::bank("de")->validate("70169464"); //true
+v::bank("de")->validate("12345"); //false
+```
+
+These country codes are supported:
+
+ * "de" (Germany)
+
+This validator needs `malkusch/bav` as dependency.
```

Comment:
This is a dependency of `de` only, so I think you should put it on the same line of `de`.